### DISCUSSION TITLE
Only translate un-translated keys

### DISCUSSION THREAD

**Discussion by colivinggit** [bernskiold/laravel-autotranslate#14] [Ideas]
Make sure that when looping through a json file, to only try to translate the values where the key == value (un-translated).
When I tried to translate the files, I felt that it was going through all keys.

**Comment by ErikBernskiold** [comment]
That's what we do. 😄 

https://github.com/bernskioldmedia/laravel-autotranslate/blob/5b59ca9c9362667968128c9ae4efdaa3eec59d61/src/TranslateStrings.php#L61-L64

**Reply by colivinggit** [reply]
Perfect. thanks for your reply.
